### ISSUE TITLE
Use !important UA stylesheet rule to override display property for never-rendered elements

### ISSUE THREAD

**Issue by AmeliaBR** [w3c/svgwg#192]
We currently have a lot of repeated prose in SVG 2 about how the `display` property doesn't apply to certain elements (the [never-rendered elements](https://svgwg.org/svg2-draft/render.html#TermNeverRenderedElement), in the terminology I added to the rendering chapter).

As I was looking up better ways to define the behavior of `<symbol>` from a Shadow DOM perspective (i.e., never-rendered directly, but rendered when re-used), I came upon the solution that it can be defined as an `!important` rule in the user agent stylesheet.

As per [CSS Cascading and Inheritance](https://www.w3.org/TR/css-cascade-4/#importance):

> User agent style sheets may also contain !important declarations. These override all author and user declarations.

In other words, instead of saying that `display` doesn't apply to these elements, we can just say that, no matter what CSS rules you apply elsewhere, a user agent `display: none !important` rule overrides them.

The net behavior is the same, but it's more logically consistent with CSS in general.  The only user-facing change is that if you `getComputedStyle()` for a never-rendered SVG element, you would now explicitly get `display: none` returned.

**Comment by nikosandronikos** [comment]
Yeah, using `display: none !important` to describe behaviour of never-rendered elements would certainly make the text more succinct. Is any clarification needed for paint servers? To describe that they can still be referenced and used to fill an object? Or does that go without saying?

**Comment by AmeliaBR** [comment]
I think we'll still need some clarification, so we won't be able to get rid of all the prose on the matter, but I don't think anything will need to be changed, either.
